packages/server/src/tests/globalSetup.ts

```diff
@@ -0,0 +1,26 @@
+import { GenericContainer, Wait } from "testcontainers"
+
+export default async function setup() {
```

Comment:
Is there any difference with the setup/teardown in backend-core? Should we reuse it?

Reply:
Nah, they're identical. I'll look in to a way to unify them.